olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -485,74 +482,63 @@ spec:
-    - name: IMAGE_DISPATCHER_IMAGE
-      image: registry.svc.ci.openshift.org/openshift/knative-v0.16.0:knative-eventing-channel-dispatcher
-    - name: IMAGE_KN_CLI_ARTIFACTS
-      image: registry.svc.ci.openshift.org/openshift/knative-v0.16.1:kn-cli-artifacts
-    - name: IMAGE_storage-version-migration__migrate
-      image: registry.svc.ci.openshift.org/openshift/knative-v0.16.0:knative-serving-storage-version-migration
-    - name: IMAGE_storage-version-migration-eventing__migrate
-      image: registry.svc.ci.openshift.org/openshift/knative-v0.16.0:knative-eventing-storage-version-migration
+  - name: "IMAGE_DISPATCHER_IMAGE"
+    image: "registry.svc.ci.openshift.org/openshift/knative-v0.16.0:knative-eventing-channel-dispatcher"
+  - name: "IMAGE_KN_CLI_ARTIFACTS"
+    image: "registry.svc.ci.openshift.org/openshift/knative-v0.16.1:kn-cli-artifacts"
+  - name: "IMAGE_autoscaler"
+    image: "registry.svc.ci.openshift.org/openshift/knative-v0.16.0:knative-serving-autoscaler"
+  - name: "IMAGE_imc-dispatcher__dispatcher"
+    image: "registry.svc.ci.openshift.org/openshift/knative-v0.16.0:knative-eventing-channel-dispatcher"
+  - name: "IMAGE_3scale-kourier-gateway"
+    image: "docker.io/maistra/proxyv2-ubi8:1.1.0"
+  - name: "IMAGE_PING_IMAGE"
+    image: "registry.svc.ci.openshift.org/openshift/knative-v0.16.0:knative-eventing-ping"
+  - name: "IMAGE_APISERVER_RA_IMAGE"
+    image: "registry.svc.ci.openshift.org/openshift/knative-v0.16.0:knative-eventing-apiserver-receive-adapter"
+  - name: "IMAGE_BROKER_FILTER_IMAGE"
+    image: "registry.svc.ci.openshift.org/openshift/knative-v0.16.0:knative-eventing-broker-filter"
+  - name: "IMAGE_imc-controller__controller"
+    image: "registry.svc.ci.openshift.org/openshift/knative-v0.16.0:knative-eventing-channel-controller"
+  - name: "IMAGE_mt-broker-filter__filter"
+    image: "registry.svc.ci.openshift.org/openshift/knative-v0.16.0:knative-eventing-mtbroker-filter"
+  - name: "IMAGE_eventing-webhook__eventing-webhook"
+    image: "registry.svc.ci.openshift.org/openshift/knative-v0.16.0:knative-eventing-webhook"
+  - name: "IMAGE_queue-proxy"
+    image: "registry.svc.ci.openshift.org/openshift/knative-v0.16.0:knative-serving-queue"
+  - name: "IMAGE_activator"
+    image: "registry.svc.ci.openshift.org/openshift/knative-v0.16.0:knative-serving-activator"
+  - name: "IMAGE_eventing-controller__eventing-controller"
+    image: "registry.svc.ci.openshift.org/openshift/knative-v0.16.0:knative-eventing-controller"
+  - name: "IMAGE_controller"
+    image: "registry.svc.ci.openshift.org/openshift/knative-v0.16.0:knative-serving-controller"
+  - name: "IMAGE_mt-broker-controller__mt-broker-controller"
+    image: "registry.svc.ci.openshift.org/openshift/knative-v0.16.0:knative-eventing-mtchannel-broker"
+  - name: "IMAGE_broker-controller__broker-controller"
+    image: "registry.svc.ci.openshift.org/openshift/knative-v0.15.2:knative-eventing-channel-broker"
```

Comment:
I miss my contextual comments for the three 0.15 images :-(

Reply:
They're in the list managed by humans though :thinking: